Api/Data/SourcesInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Ampersand\DisableStockReservation\Api\Data;
+
+/**
+ * Interface SourcesInterface
+ * @package Ampersand\DisableStockReservation\Api\Data
+ */
+interface SourcesInterface
+{
+    /**
+     * @return array|null
+     */
+    public function getSources();
```

Comment:
Can you add typehinting and return types to all methods? So `getSources() ?array`, `setSources(array $source) : SourcesInterface` etc ... Shouldn't this be a string however? You'd be setting and getting a JSON blob